contents/docs/product-analytics/web-vitals.mdx

```diff
@@ -45,4 +45,28 @@ To create your own dashboard from a template:
     videoDark="https://res.cloudinary.com/dmukukwp6/video/upload/v1725633861/posthog.com/contents/dark.mp4"
     alt="Popover icon" 
     classes="rounded"
-/>
\ No newline at end of file
+/>
+
+## Events captured
+
+The SDK groups available metrics in order to reduce the cost. That means each `$web_vitals` event carries one or more metric result. By default we wait up to 5 seconds in order to try and catch all four metrics in one event. See the section on configuring web vitals capture below for how to change this behaviour.
```

Comment:
Wait up to 5 seconds from what point?  After the last one is triggered / after the page is loaded / both?

Reply:
Ah, both... yep. 

load the page
start a timer
gather metrics
send metrics